lib/ext/adviser.rb

```diff
@@ -0,0 +1,19 @@
+require 'csv'
+
+module AdviserExtension
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def to_csv(_ = {})
+      CSV.generate do |csv|
+        csv << ['Ref. Number', 'Name', 'Firm']
+
+        find_each do |adviser|
+          csv << [adviser.reference_number, adviser.name, adviser.firm.registered_name]
+        end
+      end
+    end
+  end
+end
+
+Adviser.send(:include, AdviserExtension)
```

Comment:
@tomas-stefano @willhall88 

My question is... what's the best way to go about this? There has been varying expressions of opinions but there's no definite route to moving this implementation forward.